library/junos_install_config

```diff
@@ -165,6 +165,16 @@ options:
             - Provide a confirm in minutes to the commit of the configuration
         required: false
         default: None
+    ignore_warning:
+        description:
+            - A boolean, string or list of string.
+          If the value is True, it will ignore all warnings regardless of the
+          warning message. If the value is a string, it will ignore
+          warning(s) if the message of each warning matches the string. If
+          the value is a list of strings, ignore warning(s) if the message of
+          each warning matches at least one of the strings in the list.
```

Comment:
@vnitinv Yes. Agreed. If we need ignore_warning support for diff, we can add it at a future point.